contracts/ComptrollerStorage.sol

```diff
@@ -118,10 +118,13 @@ contract ComptrollerStorage {
     // No collateralFactorMantissa may exceed this value
     uint256 internal constant MAX_COLLATERAL_FACTOR_MANTISSA = 0.9e18; // 0.9
 
+    /// @notice Flag indicating whether forced liquidation enabled for a market
+    mapping(address => bool) public isForcedLiquidationEnabled;
+
```

Comment:
Maybe we could put the new var before the constants block or move the constants block to the top of the file so that we have some separation between these?

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/305/commits/cbd9b18a99c4e1f92bf9404e88fceb8ebc36d55f